_quarto-development.yml

```diff
@@ -0,0 +1,8 @@
+website:
+  title: Biomarker Analysis Catalog - Dev
+  navbar:
+    right:
+      - text: "Dev"
```

Comment:
@danielinteractive Should the drop-down in the navbar (for switching versions) use `Dev` and `Stable` names?

Reply:
is ok!